src/universal/redux/storageMerger.js

```diff
@@ -0,0 +1,21 @@
+import jwtDecode from 'jwt-decode';
+
+export default function merger(initialState, persistedState) {
+  const whitelist = new Set(['authToken', 'cashay', 'notifications']);
```

Comment:
will do. for the second part, after some thought i kinda like sticking with the whitelist. graphql got me hooked on explicit includes.